res/test/karma.conf.js

```diff
@@ -24,14 +28,14 @@ module.exports = function(config) {
       cache: true,
       module: webpackConfig.webpack.module,
       resolve: webpackConfig.webpack.resolve,
-      plugins: [
-        new webpack.ResolverPlugin(
-          new webpack.ResolverPlugin.DirectoryDescriptionFilePlugin(
-            'bower.json'
-            , ['main']
-          )
-        )
-      ]
+//      plugins: [
```

Comment:
It should be either removed (prefrerably) or explained why it is commented.

Reply:
that's done!